### PR TITLE
Add live-1 migration checklist first draft

### PR DESCRIPTION
**Pull request by ollieanwyll**
This PR connects to the issue ministryofjustice/cloud-platform#662

If merged this PR will add a section in the guide that covers the process of migrating an application from `live-0` to `live-1`.

This guide assumes the guides for environment and ECR generation have been updated for `live-1`.

### CODE REVIEW

File: source/documentation/other-topics/live1-migration-guide.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+Kubernetes provides a [brief guide](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#set-the-kubeconfig-environment-variable) on how to set up `kubectl` to use multiple config files simultaneously.
+
+You should now be able to switch contexts between the `live-0` and `live-1` clusters.
+
+### Generating a new environment
+
+Start by following the guide to generate a new environment, this follows the same process as was followed for `live-0`, and you should use the same details as you did for your environment then.
+
+[Enviroment generation guide.](env-create)
```
Comment (digitalronin) [inline source/documentation/other-topics/live1-migration-guide.md]: The target for this link should be `tasks.html#create-an-environment`

File: source/documentation/other-topics/live1-migration-guide.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+Run a `kubectl get namespaces` to check your environment has been successfully created.
+
+### Generating a new ECR repository
+
+Once you've generated a new environment in the `live-1` cluster, you will need to generate a new ECR repository for your application to be pushed to. 
+
+The reason why the previous ECR repo can't be used is due to the new `live-1` cluster being hosted in a separate AWS account. 
+
+If you need reminding of the ECR creation process, please see the [user documentation](ecr-setup).
```
Comment (digitalronin) [inline source/documentation/other-topics/live1-migration-guide.md]: The target for this link should be `tasks.html#creating-an-ecr-repository`

File: source/documentation/other-topics/live1-migration-guide.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+|----------|:-------------:|
+| `AWS_DEFAULT_REGION` |  The default region will now be `eu-west-2`. |
+| `AWS_ACCESS_KEY_ID` | The access key can be found in the secret created by the ECR generation. This requires base64 decoding.   |
+| `AWS_SECRET_ACCESS_KEY` |  The secret key can be found in the secret created by the ECR generation. This requires base64 decoding. |
+| `ECR_ENDPOINT` |    The ECR endpoint for all repos in `live-1` is `754256621582.dkr.ecr.eu-west-2.amazonaws.com`   |
+| `K8S_CLUSTER_CERT` |  The cert is found in the `circleci-token` secret and does not need base64 decoding. |
+| `K8S_CLUSTER_NAME` |    The cluster name is `live-1.cloud-platform.service.justice.gov.uk`  |
+| `K8S_NAMESPACE` |  This variable should be equal to the name of your namespace. |
+| `K8S_TOKEN` |    The token is found in the `circleci-token` secret and needs base64 decoding.   |
+
```
Comment (digitalronin) [inline source/documentation/other-topics/live1-migration-guide.md]: Is the `circleci-token` secret already available in the user's namespace, at this point?

Comment (digitalronin) [reply source/documentation/other-topics/live1-migration-guide.md]: For both `K8S_CLUSTER_CERT` and `K8S_TOKEN`, you've said the value is in the `circleci-token` secret. Is that correct?

File: source/documentation/other-topics/live1-migration-guide.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+### Deleting your Live-0 deployment
+
+The last thing you will need to do is to delete your application from the `live-0` cluster. Please action the 3 steps below:
+
+1. Run `helm delete --purge` or `kubectl delete` reversing the deployment commands.
+
+2. Delete all Terraform files in `resources/` except for main.tf - this will make Terraform delete all AWS resources on the next pipeline run
+
+3. Delete the `namespace` folder, which will remove any leftovers.
+
```
Comment (digitalronin) [inline source/documentation/other-topics/live1-migration-guide.md]: I would suggest adding a link to `archive.html#cleaning-up` (which also needs an update as per issue #45, but that's a separate task).

File: source/documentation/other-topics/live1-migration-guide.md (hunk 3 above)
Comment (ollieanwyll) [reply source/documentation/other-topics/live1-migration-guide.md]: It's not, it's something I added by mistake.
I've amended the guide to now reference the `default-token`.

**Pull request merged**